projects/packages/jetpack-mu-wpcom/src/features/launchpad/class-launchpad-task-lists.php

```diff
@@ -448,16 +454,57 @@ public function mark_task_complete( $task_id ) {
 			return false;
 		}
 
-		$key              = $this->get_task_key( $task );
-		$statuses         = get_option( 'launchpad_checklist_tasks_statuses', array() );
-		$statuses[ $key ] = true;
-		$result           = update_option( 'launchpad_checklist_tasks_statuses', $statuses );
+		$key    = $this->get_task_key( $task );
```

Comment:
shouldn't the `mark_task_complete` function leverage the [`wpcom_mark_launchpad_task_complete`](https://github.com/Automattic/jetpack/pull/31121/files#diff-3e92c2cff4e2dd4ec2e0ef889cac1c73a78275f711d0cd9b4e419b7753de513eR217) function added in [this PR](https://github.com/Automattic/jetpack/pull/31121)?
I know you would have to wait until that's merged but I think it will avoid code duplication.

Reply:
Based on that discussion, I'm going to move this back into In Progress for now.